### DISCUSSION TITLE
how can I setup a sub folder as project and ignore partner project ?

### DISCUSSION THREAD

**Discussion by eggcaker** [bbatsov/projectile#1725] [Q&A] [answered]
I have a project in git, it's huge project and has lot of files. I don't want to projectile to track ,instead I want make it's subfolder as a project. but problem is every time i open a file inside sub folder it auto switch to partner project . I tried remove partner project from known projects

**Comment by cfclrk** [comment, marked as answer]
The simplest thing you could do is add a `.projectile` file in the subdirectory. Just this should be enough:

```
cd subdir
touch .projectile
```
